contracts/dev-utils/contracts/src/OrderValidationUtils.sol

```diff
@@ -133,6 +133,18 @@ contract OrderValidationUtils is
             fillableTakerAssetAmount
         ) == OrderTransferResults.TransfersSuccessful ? fillableTakerAssetAmount : 0;
 
+        if (!_isAssetDataValid(order.takerAssetData)) {
+            fillableTakerAssetAmount = 0;
+        }
+
+        if (order.takerFee != 0 && !_isAssetDataValid(order.takerFeeAssetData)) {
+            fillableTakerAssetAmount = 0;
+        }
+
+        if (orderInfo.orderStatus == LibOrder.OrderStatus.CANCELLED) {
```

Comment:
Is there any reason this shouldn't be `orderInfo.orderStatus != LibOrder.OrderStatus.FILLABLE`?

Reply:
Nope, good call.